examples/user_guide/04-Dimensioned_Containers.ipynb

```diff
@@ -97,11 +99,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Exploring a parameter space using widgets is one of the most flexible approaches but as we noted above it also makes it difficult to compare between different parameter values. HoloViews therefore supplies several container objects, which behave similarly to a ``HoloMap`` but have a visual representation:\n",
+    "Exploring a parameter space using widgets is one of the most flexible approaches but as we noted above it also makes it difficult to compare between different parameter values. HoloViews therefore supplies several container objects, which behave similarly to a ``HoloMap`` but have a different visual representation:\n",
```

Comment:
I think the idea here was not that the visual representation is different, but that there was one at all.  HoloMaps simply punt on the idea of representing the additional dimensions visually, instead giving a widget so that the user is queried about which value they want to see.  Other containers actually show values along that dimension simultaneously.  Maybe you can see how to re-word it so that it would be both correct and clear?